src/entities/export.ts

```diff
@@ -5,10 +5,71 @@ import db from "../db";
 import * as schema from "../schema";
 import { serializeAccount } from "./account";
 import { serializeList } from "./list";
-import { getPostRelations, serializePost } from "./status";
+import { getPostRelations } from "./status";
 
+const homeUrl = process.env["HOME_URL"] || "http://localhost:3000";
+
+export const serializePost = (post: Post, actor: {id: ActorIdType}) => {
+  const note = {
+    "@context": "https://www.w3.org/ns/activitystreams",
+    id: new URL(post.iri),
+    type: "Note",
+    summary: post.summary || null,
+    inReplyTo: post.replyTargetId ? new URL(`${homeUrl}/posts/${post.replyTargetId}`) : null,
+    published: post.published?.toISOString() || new Date().toISOString(),
+    url: post.url ? new URL(post.url) : new URL(`${homeUrl}/posts/${post.id}`),
+    attributedTo: new URL(`${homeUrl}/accounts/${actor.id}`),
+    to: [new URL("https://www.w3.org/ns/activitystreams#Public")],
+    cc: [new URL(`${homeUrl}/accounts/${actor.id}/followers`)],
+    sensitive: post.sensitive,
+    atomUri: new URL(post.iri),
+    inReplyToAtomUri: post.replyTargetId ? new URL(`${homeUrl}/posts/${post.replyTargetId}`) : null,
+    conversation: `tag:${new URL(homeUrl).hostname},${post.published?.toISOString()}:objectId=${post.id}:objectType=Conversation`,
+    content: post.contentHtml || post.content || "",
+    contentMap: {
+      en: post.contentHtml || post.content || "",
```

Comment:
`contentMap` is not going to be in English, we should check the original post for the language.